x-pack/plugins/maps/public/components/map/mb/view.js

```diff
@@ -85,7 +137,7 @@ export class MBMapContainer extends React.Component {
       }
     }
 
-    const layer = this._getLayer(targetFeature.layer.id);
+    const layer = this._getLayerByMbLayerId(targetFeature.layer.id);
```

Comment:
Why was this changed? How does this PR effect tooltip feature layer id?

Reply:
not in particular. but this rename makes the distinction more clear between the different layer-ids.